web/packages/teleport/src/Discover/SelectResource/resources.tsx

```diff
@@ -85,6 +85,15 @@ export const SERVERS: ResourceSpec[] = [
     event: DiscoverEventResource.Ec2Instance,
     nodeMeta: { location: ServerLocation.Aws },
   },
+  {
+    name: 'Connect My Computer',
+    kind: ResourceKind.ConnectMyComputer,
+    keywords: baseServerKeywords + 'connect my computer',
+    icon: 'Laptop',
+    event: DiscoverEventResource.Server,
```

Comment:
I asked Xin if we want to count this as a separate event. For now I think I'm going to leave it as it is because otherwise I'd have to make sure that the changes are first deployed in Cloud and there's not a lot of time before 14.2.

If we end up counting Connect My Computer as a server for a week or two, it shouldn't hurt us. And its usage is tracked through Connect anyway.